mutters-core/src/main/java/com/rabidgremlin/mutters/core/MatcherScores.java

```diff
@@ -0,0 +1,69 @@
+/* Licensed under Apache-2.0 */
+package com.rabidgremlin.mutters.core;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.SortedMap;
+import java.util.TreeMap;
+
+/**
+ * Holds the matching scores generated by an IntentMatcher.
+ * 
+ * @author rabidgremlin
+ *
+ */
+public class MatcherScores
```

Comment:
would be good if this class had ```toString``` implemented

Reply:
it could also have some handy methods e.g. ```getBestScore```

Reply:
Implemented.